kie-maven-plugin/pom.xml

```diff
@@ -116,6 +72,12 @@
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.eclipse.sisu</groupId>
+          <artifactId>org.eclipse.sisu.inject</artifactId>
+        </exclusion>
```

Comment:
I am not 100% sure about this exclusion. It is needed to make the `takari-maven-plugin` lifecycle work and the tests are running fine as well. @mariofusco do you know if the `org.eclipse.sisu.inject` could be needed in some case not covered by the tests?